protoc-gen-go-grpc/pkg.yaml

```diff
@@ -9,17 +11,19 @@ steps:
         sha512: "{{ .protoc_gen_go_grpc_sha512 }}"
     prepare:
       - |
-        tar -xzf grpc-go.tar.gz --strip-components=1
+        mkdir -p /tmp/src
+        tar -xzf grpc-go.tar.gz --strip-components=1 -C /tmp/src
```

Comment:
let's add a `[TODO]` here on the `network` - whether we actually merge `/tmp/build` between stages? it's easy to do so, I don't think we use multiple steps anywhere for real

Reply:
TODO for bldr basically?

Reply:
I can make a quick fix in `bldr`, but my point was to revert these hacks. I don't think we ever need this `/tmp/build/0` and `/tmp/build/1`, I don't see value in it... Thoughts?

Reply:
Yeah, if nothing else needs this multi-step, then perhaps it'd be nice if you make everything go into a single directory. But network opt should only be applied to a section immediately after it